src/main/webapp/app/exercises/file-upload/manage/file-upload-exercise-update.component.html

```diff
@@ -64,19 +64,39 @@ <h2 *ngIf="!fileUploadExercise.id" id="jhi-file-upload-exercise-heading-create"
                     ></jhi-date-time-picker>
                     <span *ngIf="fileUploadExercise.assessmentDueDateError" class="invalid-feedback">{{ 'artemisApp.exercise.assessmentDueDateError' | translate }}</span>
                 </div>
-                <div class="form-group">
-                    <label class="form-control-label" jhiTranslate="artemisApp.exercise.maxScore" for="field_maxScore">Max Score</label>
-                    <input
-                        required
-                        type="number"
-                        class="form-control"
-                        [pattern]="maxScorePattern"
-                        min="0"
-                        max="9999"
-                        name="maxScore"
-                        id="field_maxScore"
-                        [(ngModel)]="fileUploadExercise.maxScore"
-                    />
+                <div class="row">
+                    <div class="col">
+                        <div class="form-group">
+                            <label class="form-control-label" jhiTranslate="artemisApp.exercise.points" for="field_points">Points</label>
+                            <input
+                                required
+                                type="number"
+                                class="form-control"
+                                [pattern]="maxScorePattern"
+                                min="0"
+                                max="9999"
+                                name="maxScore"
```

Comment:
Shouldn't be the name set to points?

```suggestion
                                name="points"
```